src/scene/morph-instance.js

```diff
@@ -170,25 +174,47 @@ class MorphInstance {
         return clone;
     }
 
+    _getWeightIndex(key) {
+        const index = this._weightMap.get(key);
+        if (index) {
+            return this._weights[index];
+        }
+        Debug.error('Attempting to retrieve a morph target weight using a key that doesn\'t exist.');
```

Comment:
make the missing key part of the message, so the user knows what is missing,

Reply:
Done